docs/notebooks/keras_wrapper.ipynb

```diff
@@ -0,0 +1,800 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using wrappers for Gensim models for working with Keras"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is about using gensim models as a part of your Keras models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The wrappers available (as of now) are :\n",
+    "* Word2Vec (uses the function ```get_embedding_layer``` defined in  ```gensim.models.keyedvectors```)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Word2Vec"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use Word2Vec, we import the corresponding module."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Using TensorFlow backend.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from gensim.models import word2vec"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we create a dummy set of sentences to train the Word2Vec model associated with the wrapper."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "sentences = [\n",
+    "    ['human', 'interface', 'computer'],\n",
+    "    ['survey', 'user', 'computer', 'system', 'response', 'time'],\n",
+    "    ['eps', 'user', 'interface', 'system'],\n",
+    "    ['system', 'human', 'system', 'eps'],\n",
+    "    ['user', 'response', 'time'],\n",
+    "    ['trees'],\n",
+    "    ['graph', 'trees'],\n",
+    "    ['graph', 'minors', 'trees'],\n",
+    "    ['graph', 'minors', 'survey']\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, we call the wrapper and pass appropriate parameters."
```

Comment:
What wrapper?

Here and below, the text refers some "wrapper", but I see no wrapper.